examples/simple/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/newrelic/opentelemetry-exporter-go/examples
+
+go 1.15
+
+require (
+	github.com/newrelic/newrelic-telemetry-sdk-go v0.5.1
+	github.com/newrelic/opentelemetry-exporter-go v0.14.0
+	go.opentelemetry.io/otel v0.15.0
+	go.opentelemetry.io/otel/exporters/stdout v0.15.0
```

Comment:
This is an unused dependency. Try running `go mod tidy` in the same directory - that should clean things up.

Reply:
Fixed, thanks!